<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMissingMovies</class>
 <widget class="QWidget" name="CMissingMovies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Missing Movies</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="dataSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Movie List File:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QLabel" name="serversLabel">
         <property name="text">
          <string>Servers:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="5">
        <widget class="QTreeView" name="servers">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="NSABUtils::CPathBasedDelayLineEdit" name="listFile"/>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="listFileBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="EmbySync.qrc">
           <normaloff>:/resources/open.png</normaloff>:/resources/open.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="summaryLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Summary:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NSABUtils::CPathBasedDelayLineEdit</class>
   <extends>QLineEdit</extends>
   <header>SABUtils/DelayLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listFile</tabstop>
  <tabstop>listFileBtn</tabstop>
  <tabstop>servers</tabstop>
 </tabstops>
 <resources>
  <include location="EmbySync.qrc"/>
 </resources>
 <connections/>
</ui>
